deploy/operator.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             privileged: false
             readOnlyRootFilesystem: true
             runAsNonRoot: true
-            runAsUser: 1001
+            runAsUser: 1000550000
```

Comment:
same comment about user

Reply:
That's a question for @bluzarraga, it's mainly to avoid linting error from case bundle travis build

Reply:
there will be a linting error if it's not included, so it's a matter of whether we want it explicitly set or if we want to override it and let ocp assign the user